### ISSUE TITLE
Learning rate schedule for trainer

### ISSUE THREAD

**Issue by marvinschmitt** [stefanradev93/BayesFlow#4]

**Comment by marvinschmitt** [comment]
The trainer interface passes `learning_rate` to the tensorflow optimizer. So learning rate schedules can already be used with the trainer. 

We are keeping this issue open as a reminder to discuss:

- Default learning rates
- Default learning rate scheduler (?)

**If any further functionality is desired, please issue a feature request.**
